semantic_release/hvcs.py

```diff
@@ -176,8 +196,8 @@ def get_release(cls, owner: str, repo: str, tag: str) -> int:
         :return: ID of found release
         """
         response = requests.get(
-            f"{Github.API_URL}/repos/{owner}/{repo}/releases/tags/{tag}",
-            auth=Github.auth(),
+            f"{Github.api_url()}/repos/{owner}/{repo}/releases/tags/{tag}",
+            headers={"Authorization": "token {}".format(Github.token())},
```

Comment:
Please change this back to use `auth=Github.auth()`

Reply:
derp, sorry, I knew something was a little funny here.

Fixed.
However, the test from #296 (`GithubReleaseTests.test_should_post_changelog_using_netrc`) seems to fail for me locally both on trunk and on this branch; not sure if it will pass here.

Reply:
And let me know if you have a preferred / better way to implement this; happy to rework it some if needed.